File: config/editor/src/app/definition/extentsionBoard/ExtensionBoardsConfig.ts
import {Expose} from "class-transformer";

export class ExtensionBoardsConfig {
    vendor: string;

    @Expose({name: "input_board"})
    inputBoard: string;

    @Expose({name: "output_board"})
    outputBoard: string;

    constructor(vendor: string, inputBoard: string, outputBoard: string) {
        this.vendor = vendor;
        this.inputBoard = inputBoard;
        this.outputBoard = outputBoard;
    }
}

File: config/config-sbc2ha/src/main/java/com/dfi/sbc2ha/config/sbc2ha/definition/extentsionBoard/ExtensionBoardsConfig.java
package com.dfi.sbc2ha.config.sbc2ha.definition.extentsionBoard;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

@Data
public class ExtensionBoardsConfig {
    String vendor;
    @JsonProperty("input_board")
    String inputBoard;
    @JsonProperty("output_board")
    String outputBoard;
}
